### PR TITLE
interepolation->interpolation

### PR DESCRIPTION
**Pull request by tkoyama010**
See https://github.com/pyvista/pyvista/pull/3403

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -19365,6 +19365,7 @@ intereference->interference
 intereferences->interferences
 interelated->interrelated
 interelaved->interleaved
+interepolation->interpolation
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:19368]:
```suggestion
interepolate->interpolate
interepolated->interpolated
interepolates->interpolates
interepolating->interpolating
interepolation->interpolation
```

Comment (vikivivi) [reply codespell_lib/data/dictionary.txt:19368]: I did not suggest `interpolations` due to 0 hit.

* https://github.com/search?q=interepolate&type=code  11
* https://github.com/search?q=interepolated&type=code  21
* https://github.com/search?q=interepolates&type=code  5
* https://github.com/search?q=interepolating&type=code  3
* https://github.com/search?q=interepolation&type=code  178
* https://github.com/search?q=interepolations&type=code  0

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Review by peternewman — APPROVED** [review]
LGTM thanks.

**Comment by peternewman** [comment]
Force merging as Python 3.10 test is missing for some reason.

**Pull request merged**